packages/ui-components/style/toolbar.css

```diff
@@ -101,6 +102,12 @@ button.jp-ToolbarButtonComponent .jp-ToolbarButtonComponent-label {
   font-family: var(--jp-ui-font-family);
 }
 
+.jp-ToolbarLabelComponent {
+  background-color: var(--jp-brand-color1);
+  color: var(--jp-ui-inverse-font-color1);
+  font-size: var(--jp-ui-font-size0);
```

Comment:
One tiny addition:

```suggestion
  font-size: var(--jp-ui-font-size0);
  user-select: none;
```

To avoid inconsistent look when user accidentally highlights it:

![Screenshot from 2023-08-29 21-33-07](https://github.com/jupyterlab/jupyterlab/assets/5832902/f91a2d69-6ee6-49de-96d7-0d8902a08298)